file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [11-May-2025 03:42:13 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT *  FROM wp_a2g_tm_kategorie</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT * FROM `wp_a2g_tm_kategorie` | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_a2g_tm_kategorie
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('SELECT *  FROM ...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('SELECT *  FROM ...')
#3 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT *  FROM ...')
#4 /wordpress/wp-content/plugins/ada2go-text-modules/ada2go-text-modules.php(103): wpdb->get_results('SELECT *  FROM ...')
#5 /wordpress/wp-includes/class-wp-hook.php(324): a2g_tm_activate(false)
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_ada2go...', false)
#9 /internal/eval.php(20): activate_plugin('ada2go-text-mod...')
#10 {main}</pre>
 for query SELECT *  FROM wp_a2g_tm_kategorie made by activate_plugin, do_action('activate_ada2go-text-modules/ada2go-text-modules.php'), WP_Hook->do_action, WP_Hook->apply_filters, a2g_tm_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[11-May-2025 03:42:13 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT *  FROM wp_a2g_tm_text</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT * FROM `wp_a2g_tm_text` | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_a2g_tm_text
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('SELECT *  FROM ...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('SELECT *  FROM ...')
#3 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT *  FROM ...')
#4 /wordpress/wp-content/plugins/ada2go-text-modules/ada2go-text-modules.php(104): wpdb->get_results('SELECT *  FROM ...')
#5 /wordpress/wp-includes/class-wp-hook.php(324): a2g_tm_activate(false)
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_ada2go...', false)
#9 /internal/eval.php(20): activate_plugin('ada2go-text-mod...')
#10 {main}</pre>
 for query SELECT *  FROM wp_a2g_tm_text made by activate_plugin, do_action('activate_ada2go-text-modules/ada2go-text-modules.php'), WP_Hook->do_action, WP_Hook->apply_filters, a2g_tm_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[11-May-2025 03:42:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT *  FROM wp_a2g_tm_kategorie</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT *  FROM wp_a2g_tm_kategorie | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_a2g_tm_kategorie.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT *  FROM ...')
#2 /wordpress/wp-content/plugins/ada2go-text-modules/ada2go-text-modules.php(103): wpdb->get_results('SELECT *  FROM ...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): a2g_tm_activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_ada2go...', false)
#7 /internal/eval.php(20): activate_plugin('ada2go-text-mod...')
#8 {main}</pre>
 for query SELECT *  FROM wp_a2g_tm_kategorie made by activate_plugin, do_action('activate_ada2go-text-modules/ada2go-text-modules.php'), WP_Hook->do_action, WP_Hook->apply_filters, a2g_tm_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[11-May-2025 03:42:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT *  FROM wp_a2g_tm_text</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT *  FROM wp_a2g_tm_text | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_a2g_tm_text.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT *  FROM ...')
#2 /wordpress/wp-content/plugins/ada2go-text-modules/ada2go-text-modules.php(104): wpdb->get_results('SELECT *  FROM ...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): a2g_tm_activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_ada2go...', false)
#7 /internal/eval.php(20): activate_plugin('ada2go-text-mod...')
#8 {main}</pre>
 for query SELECT *  FROM wp_a2g_tm_text made by activate_plugin, do_action('activate_ada2go-text-modules/ada2go-text-modules.php'), WP_Hook->do_action, WP_Hook->apply_filters, a2g_tm_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[11-May-2025 03:42:30 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT *  FROM wp_a2g_tm_kategorie</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT *  FROM wp_a2g_tm_kategorie | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_a2g_tm_kategorie.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT *  FROM ...')
#2 /wordpress/wp-content/plugins/ada2go-text-modules/ada2go-text-modules.php(103): wpdb->get_results('SELECT *  FROM ...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): a2g_tm_activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_ada2go...', false)
#7 /internal/eval.php(20): activate_plugin('ada2go-text-mod...')
#8 {main}</pre>
 for query SELECT *  FROM wp_a2g_tm_kategorie made by activate_plugin, do_action('activate_ada2go-text-modules/ada2go-text-modules.php'), WP_Hook->do_action, WP_Hook->apply_filters, a2g_tm_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[11-May-2025 03:42:30 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT *  FROM wp_a2g_tm_text</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT *  FROM wp_a2g_tm_text | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_a2g_tm_text.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT *  FROM ...')
#2 /wordpress/wp-content/plugins/ada2go-text-modules/ada2go-text-modules.php(104): wpdb->get_results('SELECT *  FROM ...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): a2g_tm_activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_ada2go...', false)
#7 /internal/eval.php(20): activate_plugin('ada2go-text-mod...')
#8 {main}</pre>
 for query SELECT *  FROM wp_a2g_tm_text made by activate_plugin, do_action('activate_ada2go-text-modules/ada2go-text-modules.php'), WP_Hook->do_action, WP_Hook->apply_filters, a2g_tm_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #5 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"addon-vryno-for-gravity-form"},"options":{"activate":true}}) : Plugin /wordpress/wp-content/plugins/addon-vryno-for-gravity-form could not be activated – WordPress exited with no error. Sometimes, when $_SERVER or site options are not configured correctly, WordPress exits early with a 301 redirect. Inspect the "debug" logs in the console for more details.
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      ... 2 lines matching cause stack trace ...
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: Error: Plugin /wordpress/wp-content/plugins/addon-vryno-for-gravity-form could not be activated – WordPress exited with no error. Sometimes, when $_SERVER or site options are not configured correctly, WordPress exits early with a 301 redirect. Inspect the "debug" logs in the console for more details.
        at wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:119:51)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)
        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)
        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)
        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12)
  }
}

Node.js v23.11.0
